chore: introduce vitest.config.ts in each package (#12523)
owncloud/web: docs/testing/testing.md

```shell
$ pnpm test:unit
$ pnpm -r test:unit 
```

> You can also specify which package to run the test on, like so: `pnpm --filter @ownclouders/web-pkg test:unit`
> Specific test could also be run by navigating to the package name and then running `pnpm test:unit`

Reviewer: ```suggestion
You can also specify which package to run the test on, such as: `pnpm --filter @ownclouders/web-pkg test:unit`.
Specific test could also be run by navigating to the package name and then running `pnpm test:unit`.
```

> Specific test could also be run by navigating to the package name and then running `pnpm test:unit`.

Reviewer: ```suggestion
Tests can also be run by navigating to the package name and then running `pnpm test:unit`.
```
